runtime/doc/options.txt

```diff
@@ -762,13 +762,6 @@ A jump table for the options with a short description can be found at |Q_op|.
 	When the value is empty, Vi compatible backspacing is used, none of
 	the ways mentioned for the items above are possible.
 
-	For backwards compatibility with version 5.4 and earlier:
-	value	effect	~
-	  0	same as ":set backspace=" (Vi compatible)
-	  1	same as ":set backspace=indent,eol"
-	  2	same as ":set backspace=indent,eol,start"
-	  3	same as ":set backspace=indent,eol,nostop"
```

Comment:
We should add this to `news.txt`, though, in the form of a Rosetta Stone:
```
instead of
* `backspace=0` set `backspace=` (empty)
* `backspace=1` set `backspace=indent,eol`
* `backspace=2` remove (default behavior in Nvim)
* `backspace=3` set `backspace=indent,eol,nostop`